src/nvim/mapping.c

```diff
@@ -2798,15 +2820,21 @@ void modify_keymap(uint64_t channel_id, Buffer buffer, bool is_unmap, String mod
   case 0:
     break;
   case 1:
-    api_set_error(err, kErrorTypeException, e_invarg, 0);
+    api_set_error(err, kErrorTypeException, _(e_invarg), 0);
     goto fail_and_free;
   case 2:
-    api_set_error(err, kErrorTypeException, e_nomap, 0);
+    api_set_error(err, kErrorTypeException, _(e_nomap), 0);
     goto fail_and_free;
   case 5:
     api_set_error(err, kErrorTypeException,
-                  is_abbrev ? e_abbreviation_already_exists_for_str
-                            : e_mapping_already_exists_for_str, lhs.data);
+                  is_abbrev ? _(e_abbreviation_already_exists_for_str)
+                            : _(e_mapping_already_exists_for_str), lhs.data);
+    goto fail_and_free;
+    break;
+  case 6:
+    api_set_error(err, kErrorTypeException,
+                  is_abbrev ? _(e_global_abbreviation_already_exists_for_str)
+                            : _(e_global_mapping_already_exists_for_str), lhs.data);
     goto fail_and_free;
```

Comment:
API errors should not be translated. See https://github.com/neovim/neovim/issues/18128#issuecomment-1107946770